elytron/src/main/java/org/wildfly/extension/elytron/RealmParser.java

```diff
@@ -137,11 +154,11 @@
             .addChild(customRealmParser)
             .addChild(customModifiableRealmParser)
             .addChild(identityRealmParser)
-            .addChild(jdbcRealmParser_7_0)
+            .addChild(jdbcRealmParser_12_0)
```

Comment:
Instead of modifying the parser values here, the existing `realmParser_11_0` variable needs to be copied to a new `realmParser_12_0` variable and then that should be updated to reference the new `jdbcRealmParser_12_0` variable, the new `propertiesRealmParser_12_0` variable, etc.